src/fides/api/service/saas_request/override_implementations/authentication_strategy_power_reviews.py

```diff
@@ -0,0 +1,61 @@
+from typing import Dict, cast
+
+from requests import PreparedRequest, post
+
+from fides.api.common_exceptions import FidesopsException
+from fides.api.models.connectionconfig import ConnectionConfig
+from fides.api.schemas.saas.strategy_configuration import StrategyConfiguration
+from fides.api.service.authentication.authentication_strategy import (
+    AuthenticationStrategy,
+)
+
+
+class PowerReviewsAuthenticationConfiguration(StrategyConfiguration):
+    """
+    Parameters to authorize a PowerReviews connection
+    """
+
+    client_id: str
+    client_secret: str
+
+
+class PowerReviewsAuthenticationStrategy(AuthenticationStrategy):
```

Comment:
Could we have used the `oauth2_client_credentials` authentication strategy instead?

Reply:
Initially i thought we wouldn't be able, since its a cross between Oauth2 and Basic auth, but it looks like it can be done. Updating it